Sources/TuistKit/Commands/Cloud/CloudPushFrameworksCommand.swift

```diff
@@ -4,10 +4,10 @@ import TSCBasic
 import TuistSupport
 
 /// Command to cache frameworks as .xcframeworks and speed up your and others' build times.
-struct CacheCommand: ParsableCommand {
+struct CloudPopulateCacheCommand: ParsableCommand {
     static var configuration: CommandConfiguration {
-        CommandConfiguration(commandName: "cache",
-                             abstract: "Cache frameworks as .xcframeworks to speed up build times in generated projects")
+        CommandConfiguration(commandName: "populate-cache",
```

Comment:
What do you think about using `warm` instead of of `populate`? It sounds more like a technical term to me